zebra/zebra_ted.c

```diff
@@ -0,0 +1,270 @@
+/*
+ * Zebra TE data module
+ * Copyright (c) 2020 Volta Networks, Inc.
```

Comment:
Align name of new module according to the name of new message. Perhaps `link-state` is more appropriate as information send by IGP are Link State info, not the Traffic Engineering Database